### PR TITLE
Migrate Spring Cloud Functions to Python Chalice

### PR DESCRIPTION
**Pull request by pshumateEDC**
Write skeleton of Idemia Microservice in Python Chalice. Each endpoint currently contains "hello world" functionality.

### CODE REVIEW

**Review by dzaslavskiy — APPROVED** [review]

**Review by dzaslavskiy — COMMENTED** [review]
resolve merge conflict

File: pipeline/app.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+app = core.App()
+
+app_name = "give-ipp-idemia"
+pipeline_stack_name = f"{app_name}-pipeline"
+
+PipelineStack(
+    app,
+    app_name,
+    stack_name=pipeline_stack_name,
+    repo_owner="pshumateEDC",
```
Comment (folksgl) [inline pipeline/app.py:17]: This should be 18F now

File: .gitignore [hunk 2]
```diff
@@ -1,39 +1,7 @@
...
-Idemia-Microservice/target/
-!idemia-microservice/.mvn/wrapper/maven-wrapper.jar
-!**/src/main/**/target/
-!**/src/test/**/target/
-
-### AWS ###
-.aws-sam/
-packaged.yaml
-.cdk.staging
+.chalice/deployments/
```
Comment (folksgl) [inline .gitignore:1]: Please add .chalice/deployed/

File: app/.chalice/config.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "version": "2.0",
+  "app_name": "give_idemia_microservice",
```
Comment (folksgl) [inline app/.chalice/config.json:3]: This should match the naming convention identity-give-<app_name>
so here it should be "ipp-idemia"

File: app/app.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+from chalice import Chalice
+
+app = Chalice(app_name="give_idemia_microservice")
```
Comment (folksgl) [inline app/app.py:3]: See config.json comment on naming

File: pipeline/app.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+
+from pipeline_stack import PipelineStack
+
+
+app = core.App()
+
+app_name = "give-ipp-idemia"
```
Comment (folksgl) [inline pipeline/app.py:10]: See config.json comment on naming

**Review by folksgl — CHANGES_REQUESTED** [review]

**Pull request merged**